docs/source/reference/label_format.rst

```diff
@@ -1528,7 +1528,7 @@ To add a :class:`~tensorbay.label.label_mask.SemanticMask` label to one data:
  InstanceMask
 **************
 
-InstanceMask is a type of label which is usually used for semantic segmentation task.
+InstanceMask is a type of label which is usually used for instance segmentation task.
```

Comment:
commit message
`docs: fix the typo in the "InstanceMask" chapter of "label_format.rst"`